cardano-cli/src/Cardano/CLI/EraBased/Run/Genesis/CreateTestnetData.hs

```diff
@@ -273,7 +273,7 @@ runGenesisCreateTestNetDataCmd
             mapAccumM
               (\g' _ -> swap . first getVerificationKey <$> generateInsecureSigningKey g' AsDRepKey)
               g
-              [1 .. numOfStakeDelegators]
+              [1 .. numOfDRepCredentials]
```

Comment:
I was testing your code and I found a bug in my code, which higlights that I test other people's code more rigorously than my own, but well. Here is the patch I added to your PR: https://github.com/IntersectMBO/cardano-cli/pull/943/commits/0109d11a657473a781dc8c2738f0b60cf40b4ba8